pkg/v1dbschema/schema_test.go

```diff
@@ -129,50 +132,70 @@ func (t *testSchema) TestSchemaV106ToV20x(c *C) {
 	}()
 
 	// create metadata schema.
-	_, err = dbConn.ExecuteSQL(tctx, nil, cfg.Name, []string{
-		`CREATE DATABASE IF NOT EXISTS ` + cfg.MetaSchema,
-	})
+	createSchemaSQL := `CREATE DATABASE IF NOT EXISTS ` + cfg.MetaSchema
+	t.mockDB.ExpectBegin()
+	t.mockDB.ExpectExec(createSchemaSQL).WithArgs().WillReturnResult(sqlmock.NewErrorResult(nil))
+	t.mockDB.ExpectCommit()
+	_, err = dbConn.ExecuteSQL(tctx, nil, cfg.Name, []string{createSchemaSQL})
 	c.Assert(err, IsNil)
 
 	// create v1.0.6 checkpoint table.
 	createCpV106, err := ioutil.ReadFile(filepath.Join(v1DataDir, "v106_syncer_checkpoint-schema.sql"))
 	c.Assert(err, IsNil)
-	_, err = dbConn.ExecuteSQL(tctx, nil, cfg.Name, []string{
-		string(createCpV106),
-	})
+	createCpV106SQL := string(createCpV106)
+	t.mockDB.ExpectBegin()
+	t.mockDB.ExpectExec(createCpV106SQL[:10] + ".*").WithArgs().WillReturnResult(sqlmock.NewErrorResult(nil))
```

Comment:
use this `createCpV106SQL[:10] + ".*"` becasue if i pass `createCpV106SQL` to mockdb, it will return a not matched error 🤔, i don't konw why, need some help here

Reply:
maybe the regular expression engine doesn't support multi-lines matching.